man/crystal.1

```diff
@@ -0,0 +1,259 @@
+.\"Crystal Programming Language
+.Dd February 11, 2016
+.Dt CRYSTAL(1) "" "Crystal Programmers Reference Guide"
+.\".Dt CRYSTAL 1
+.Os UNIX
+.Sh NAME
+.Nm crystal
+.Nd Compiler for the statically type-checked programming language Crystal.
+.Sh SYNOPSYS
+.Nm
+.Op command
+.Op switches
+.Op program file
+.Op --
+.Op arguments
+.Sh DESCRIPTION
+Once you install the compiler you will have a crystal binary at your disposal.
+
+.Bl -tag -width "12345678" -compact
+.It Compiling 
+.Pp
+To compile and run a program in a single shot you can invoke crystal with a single filename:
+
+$ crystal some_program.cr
+
+Crystal files end with the .cr extension.
+
+Alternative you can use the run command:
+
+$ crystal run some_program.cr
+
+.It Creating an executable
+.Pp
+To create an executable use the build command:
+
+$ crystal build some_program.cr
+
+This will create a some_program file that you can execute:
+
+$ ./some_program
+
+Note: By default the generated executables are not fully optimized. 
+To turn optimizations on, use the --release flag:
+
+$ crystal build some_program.cr --release
+
+Make sure to always use --release for production-ready executables and when performing benchmarks.
+
+The reason for this is that performance without full optimizations is still pretty good and provides fast compile times, so you can use the crystal command almost as if it were an interpreter.
+
+.Sh OPTIONS
+The crystal command line accepts the following options 
+
+.Bl -tag -width "12345678" -compact
+.Pp
+.It 
+.Cm init
+.Ar TYPE
+.Ar NAME
+.Op DIR
+.Pp
+Generate a new project.
+.Pp
+TYPE is one of:
+.Bl -tag -width "12345678" -compact
+.Pp
+.It Sy lib
+creates library skeleton
+.It Sy app
+creates application skeleton
+.El
+.Bd -literal -offset
+NAME - name of project to be generated,
+eg: example
+.Pp
+DIR  - directory where project will be generated,
+default: NAME, eg: ./custom/path/example
+.Ed
+
+.Pp
+.It
+.Cm build
+.Op options
+.Op programfile
+.Op --
+.Op arguments
+.Pp
+Compile program.
+.Pp
+Options:
+.Bl -tag -width "12345678" -compact
+.Pp
+.It Fl -cross-compile flags
```

Comment:
Outdated, in general please review the available commands and options on the latest compiler.